secrets/config.pangeo-forge-staging.yaml

```diff
@@ -1,28 +1,26 @@
 fastapi:
-    ADMIN_API_KEY_SHA256: ENC[AES256_GCM,data:ywNekbjNwWU5bHPUKwy89DiMxwxlHTmTCpWVvVTtomtrEQOKf9nMs49udgOUWipBKTA2efBu5xL/1/VMnOoN0A==,iv:Mi/8qt7QtUEu+1WOxxwOIwpTSBEbbiyzj658NTtxmtU=,tag:1/FuCzwHLJxL9SDYHzkxKw==,type:str]
-    ENCRYPTION_SALT: ENC[AES256_GCM,data:7zf4yrrRV6fKv3RGrl3VcErmdo8Z/iuhoPFRBUl84HU=,iv:mRTu34oFfrGR6aWWN3cSZXytWAuyTC6tVt8XIFs44Ho=,tag:zTdc6hrYjeBTlm2VLm/FIA==,type:str]
-    PANGEO_FORGE_API_KEY: ENC[AES256_GCM,data:ykUQyFy3szpFbZLaUi0QPcylx7LE1orrndSV/b+fasU=,iv:f4mgLR9rTwB2pd/aBC4NOh2G0kLWIbFv6djLOGVJCIk=,tag:qH+yfkSJMof994cUu7SLpg==,type:str]
+    PANGEO_FORGE_API_KEY: ENC[AES256_GCM,data:xkJeBOPCKtQDq6xeSYHF+qZmlIxDoypVQhhFk98snPc=,iv:HT1eo9RrrgMZaV4TNb7o7EmudgU+r3JA5p/BV7vOAzg=,tag:WSV6xQdkg8aP5G3GjgCevA==,type:str]
```

Comment:
So this presumably contains the encrypted key that @cisaacstern generated manually somewhere. Correct?

Reply:
Correct. I manually generated this with `scripts.develop/generate_api_key.py`.